### DISCUSSION TITLE
Welcome to 30-Days-Of-CPP Discussions!

### DISCUSSION THREAD

**Discussion by subhadipbhowmik** [subhadipbhowmik/30-Days-Of-CPP#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sibam-Paul** [comment]
@subhadipbhowmik need some help in understanding the code

**Comment by Sibam-Paul** [comment]
@subhadipbhowmik Please help

**Comment by Sahil-Vaidya** [comment]
@subhadipbhowmik  check my pull request i have added day 5 content.
